src/main/java/com/minecolonies/core/entity/citizen/citizenhandlers/CitizenDiseaseHandler.java

```diff
@@ -93,7 +93,7 @@ public void tick()
             final int citizenModifier = citizen.getCitizenJobHandler().getColonyJob() == null ? 1 : citizen.getCitizenJobHandler().getColonyJob().getDiseaseModifier();
             final int configModifier = MineColonies.getConfig().getServer().diseaseModifier.get();
             if (!IColonyManager.getInstance().getCompatibilityManager().getDiseases().isEmpty() &&
-                  citizen.getRandom().nextInt(configModifier * DISEASE_FACTOR) < citizenModifier)
+                  citizen.getRandom().nextInt(configModifier * DISEASE_FACTOR/1) < citizenModifier)
```

Comment:
Curious what the division by 1 does

Reply:
Probably needed for type changing from int to a double or something

Reply:
bug, fix it! :D

Reply:
debug value derp, fixed